docs/tutorials/quickstart_cogshop_run.ipynb

```diff
@@ -0,0 +1,823 @@
+       "    <tr>\n",
+       "      <th>shop_files</th>\n",
+       "      <td>[shopfile:6eb907ff9fa34951b83065df96e3f725]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "ShopCase(space='power_ops_instances', external_id='example_case_external_id', data_record=DataRecord(version=1, last_updated_time=datetime.datetime(2024, 10, 8, 15, 34, 11, 904000, tzinfo=TzInfo(UTC)), created_time=datetime.datetime(2024, 10, 8, 15, 34, 11, 904000, tzinfo=TzInfo(UTC)), deleted_time=None), node_type=None, start_time=datetime.datetime(2023, 9, 14, 22, 0, tzinfo=TzInfo(UTC)), end_time=datetime.datetime(2023, 9, 24, 22, 0, tzinfo=TzInfo(UTC)))"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from cognite.powerops.client._generated.v1.data_classes._shop_case import ShopCase\n",
+    "\n",
+    "\n",
+    "# NB! This step is not necessary, if the upsert was successful we know the case is in CDF\n",
+    "def retrieve_shop_case(case_external_id: str) -> ShopCase:\n",
+    "    \"\"\"Retrieve a shop case from CDF\"\"\"\n",
+    "    # return powerops.v1.get(external_id)\n",
+    "    return powerops.v1.day_ahead_bid.shop_case.retrieve(external_id=case_external_id)\n",
+    "\n",
+    "\n",
+    "retrieved_shop_case = retrieve_shop_case(EXAMPLE_CASE_EXTERNAL_ID)\n",
+    "retrieved_shop_case"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d38c1534",
+   "metadata": {},
+   "source": [
+    "### Step 4: Trigger the case\n",
+    "\n",
+    "We will here directly use the new `trigger_shop_case` method on the `powerops.cogshop` client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "934e7d52",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "powerops.cogshop.trigger_shop_case(EXAMPLE_CASE_EXTERNAL_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd4b60ee",
+   "metadata": {},
+   "source": [
+    "### Step 5: View the ShopResult generated \n",
+    "\n",
+    "Explanations for (some) of the fields of the `ShopResult`: \n",
+    "\n",
+    "* External id: The external id of the `ShopResult`\n",
+    "* Case: The external id of the `ShopCase` that the instance belongs to\n",
+    "* Objective value: The objective value of the Shop execution\n",
+    "* Pre run: A file reference to the pre run yaml file\n",
+    "* Post run: A file reference to the post run yaml file\n",
+    "* Messages: A file reference to the logs generated by Shop\n",
+    "* Cplex logs: A file reference to the cplex logs generated by Shop\n",
+    "* Data record: The data record of the instance, contains `last_updated_time` and `created_time`\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5470afe6",
+   "metadata": {},
+   "source": [
+    "\n",
+    "#### 1. Querying for ShopResult instance, given their ShopCase external id\n"
```

Comment:
```suggestion
    "#### 1. Querying for ShopResult instance, given their ShopCase external id\n"
    "Note that it may take a while to run (Cog)Shop, and if the run is not completed, no results are returned"
```
Can we mention how to look at status too, or better to avoid it because it's not covered by the SDK?